spec/models/weapon_spec.rb

```diff
@@ -10,6 +10,26 @@
   it { should validate_numericality_of(:encum).is_greater_than_or_equal_to(0).is_less_than_or_equal_to(99) }
   it { should validate_numericality_of(:hp).is_greater_than_or_equal_to(0).is_less_than_or_equal_to(99) }
   it { should validate_numericality_of(:rarity).is_greater_than_or_equal_to(0).is_less_than_or_equal_to(99) }
+  it { should validate_numericality_of(:default_damage).is_greater_than_or_equal_to(0).is_less_than_or_equal_to(99) }
+  it { should validate_numericality_of(:default_crit).is_greater_than_or_equal_to(0).is_less_than_or_equal_to(99) }
   it { should have_many(:character_weapons) }
   it { should have_many(:characters).through(:character_weapons) }
+
+  describe "Odered weapons by rarity" do
+    it "should return the rarest weapon as the first element of the list" do
+      5.times { FactoryGirl.create :weapon }
+      expect(Weapon.order_by_rarity.first.rarity).to eql Weapon.maximum("rarity")
```

Comment:
I don't feel this is a good test, you are not actually testing if the other weapons are ordered, jus the first one, and you are ordering by two criteria, and the `Weapon.maximum("rarity")` is just testing for rarity